src/utils/locales.ts

```diff
@@ -0,0 +1,24 @@
+import fs from "fs";
+import { NODE_MODULES_LOCALES_PATH } from "../constants/constants";
+import log from "./log";
+
+export function copyLocales(): void {
+  const sources = [NODE_MODULES_LOCALES_PATH, `../${NODE_MODULES_LOCALES_PATH}`];
+  const targets = ["../locales", "./locales"];
+
+  const copied = sources.some(src =>
+      targets.some(target => {
+        try {
+          fs.cpSync(src, target, { recursive: true });
+          log(`Locales copied from ${src} to ${target}`);
+          return true;
+        } catch (_) {
```

Comment:
We're logging an error below if the copying task fails which is useful but here when we catch the original error we throw away the context, is it worth keeping anything so that if anyone runs into the log can understand why it has failed?